BrightID/src/actions/apps.ts

```diff
@@ -88,13 +87,18 @@ export const updateBlindSigs = (api) => async (
           console.log(`wrong signature for ${app.name} (${verification})!`);
           continue;
         }
+        // TODO: This will overwrite existing sigs if rounded timestamp does not match
+        // apps timestamps. Is this intended? Should this affect the linked state of an app?
         dispatch(
           addSig({
+            id: `${app.id}_${verification}`,
```

Comment:
you can not consider `app.id` and `verification` to be unique as apps can/should renew their verification for same `app.id` and `verification` with a different `roundedTimestamp`.